Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
+
+COPY <<EOF /etc/yum.repos.d/mongodb-org-6.0.repo
+[mongodb-org-6.0]
+name=MongoDB Repository
+baseurl=https://repo.mongodb.org/yum/redhat/8/mongodb-org/6.0/x86_64/
+gpgcheck=1
+enabled=1
+gpgkey=https://www.mongodb.org/static/pgp/server-6.0.asc
+EOF
+
+RUN microdnf install shadow-utils jq yum yum-utils &&\
+    yum -y update &&\
+    yum clean all &&\
+    microdnf clean all
+
+RUN yum install -y mongodb-atlas
```

Comment:
We missing Entripoint and exec. That leaves the option on how the command is run:

- Bash shell is running and it runs atlas-cli. That can run 
- Just atlas-cli is running without shell (non-TTY)

I will recommend to experiment with shell and learn about entry points for CLI docker images. 

Try to answer questions like:
Do we want to use docker run for every command or we start bash shell where atlas cli can run?

Reply:
I added an entrypoint for now that executes the atlas binary, as discussed still up for debate how this should be done dependant on how most people will use the image

Reply:
Sounds good. We can change that during development time.